scripts/optimize_zshrc.sh

```
@@ -0,0 +1,158 @@
+    exit 0
+fi
+
+# Ищем строку с загрузкой setup_cli_aliases.sh
+if grep -q "setup_cli_aliases.sh" "$ZSHRC_FILE"; then
+    echo "📝 Найдена загрузка setup_cli_aliases.sh"
+    echo "   Обновляю для использования тихого режима..."
+    echo ""
+
+    # Создаем временный файл для безопасного редактирования
+    TEMP_FILE=$(mktemp "${ZSHRC_FILE}.tmp.XXXXXX")
+    trap "rm -f '$TEMP_FILE'" EXIT INT TERM
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)